--- zhpev_drv.f90	2017-03-03 07:47:03.000000000 +0100
+++ zhpev_drv_tmp.f90	2017-05-02 14:54:38.591915904 +0200
@@ -1519,13 +1519,13 @@
 
 #if defined(__ELPA_2016)
      ! -> ELPA 2016.11.001_pre
-     ierr = elpa_get_communicators(ortho_comm, my_prow, my_pcol,mpi_comm_rows, mpi_comm_cols)
-     success = solve_evp_complex_1stage_double(n, n, h, size(h,1), w,  v, size(h,1), size(h,2), nb, &
-                           mpi_comm_rows, mpi_comm_cols, ortho_comm)
+     !ierr = elpa_get_communicators(ortho_comm, my_prow, my_pcol,mpi_comm_rows, mpi_comm_cols)
+     !success = solve_evp_complex_1stage_double(n, n, h, size(h,1), w,  v, size(h,1), size(h,2), nb, &
+     !                      mpi_comm_rows, mpi_comm_cols, ortho_comm)
      ! -> ELPA 2016.05.003
-     !ierr = get_elpa_row_col_comms(ortho_comm, my_prow, my_pcol,mpi_comm_rows, mpi_comm_cols)
-     !success = solve_evp_complex(n, n, h, size(h,1), w,  v, size(h,1), size(h,2), nb, &
-     !                      mpi_comm_rows, mpi_comm_cols)
+     ierr = get_elpa_row_col_comms(ortho_comm, my_prow, my_pcol,mpi_comm_rows, mpi_comm_cols)
+     success = solve_evp_complex(n, n, h, size(h,1), w,  v, size(h,1), size(h,2), nb, &
+                           mpi_comm_rows, mpi_comm_cols)
 #elif defined(__ELPA_2015)
      ierr = get_elpa_row_col_comms(ortho_comm, my_prow, my_pcol,mpi_comm_rows, mpi_comm_cols)
      ierr = solve_evp_complex(n, n, h, size(h,1), w,  v, size(h,1), size(h,2), nb, &
